tools/http-fault-injector/Azure.Sdk.Tools.HttpFaultInjector/Program.cs

```diff
@@ -71,364 +30,59 @@ public static void Main(string[] args)
+            var builder = WebApplication.CreateBuilder(args);
+            builder.WebHost.ConfigureKestrel(kestrelOptions =>
             {
-                case "f":
-                    // Full response
-                    await SendDownstreamResponse(upstreamResponse, context.Response, upstreamResponse.ContentLength, context.RequestAborted);
-                    return true;
-                case "p":
-                    // Partial Response (full headers, 50% of body), then wait indefinitely
-                    await SendDownstreamResponse(upstreamResponse, context.Response, upstreamResponse.ContentLength / 2, context.RequestAborted);
-                    await Task.Delay(Timeout.InfiniteTimeSpan, context.RequestAborted);
-                    return true;
-                case "pc":
-                    // Partial Response (full headers, 50% of body), then close (TCP FIN)
-                    await SendDownstreamResponse(upstreamResponse, context.Response, upstreamResponse.ContentLength / 2, context.RequestAborted);
-                    Close(context);
-                    return true;
-                case "pa":
-                    // Partial Response (full headers, 50% of body), then abort (TCP RST)
-                    await SendDownstreamResponse(upstreamResponse, context.Response, upstreamResponse.ContentLength / 2, context.RequestAborted);
-                    Abort(context);
-                    return true;
-                case "pn":
-                    // Partial Response (full headers, 50% of body), then finish normally
-                    await SendDownstreamResponse(upstreamResponse, context.Response, upstreamResponse.ContentLength / 2, context.RequestAborted);
-                    return true;
-                case "n":
-                    // No response, then wait indefinitely
-                    await Task.Delay(Timeout.InfiniteTimeSpan, context.RequestAborted);
-                    return true;
-                case "nc":
-                    // No response, then close (TCP FIN)
-                    Close(context);
-                    return true;
-                case "na":
-                    // No response, then abort (TCP RST)
-                    Abort(context);
-                    return true;
-                default:
-                    Console.WriteLine($"Invalid selection: {selection}");
-                    return false;
-            }
-        }
-
-        private static async Task SendDownstreamResponse(UpstreamResponse upstreamResponse, HttpResponse response, long? contentBytes, CancellationToken cancellationToken)
-        {
-            Console.WriteLine();
-
-            Log("Sending downstream response...");
-
-            response.StatusCode = upstreamResponse.StatusCode;
+                kestrelOptions.Limits.KeepAliveTimeout = keepAlive;
+            });
 
-            Log($"StatusCode: {upstreamResponse.StatusCode}");
+            builder.Services.AddControllers();
```

Comment:
Do we still need this if we don't have any controllers?  It's probably a no-op so doesn't matter, but curious about best practices.

Reply:
good catch! no we don't need it anymore